src/sentry/ingest/userreport.py

```diff
@@ -41,21 +36,12 @@ def save_userreport(project, report, start_time=None):
 
         event = eventstore.backend.get_event_by_id(project.id, report["event_id"])
 
-        # TODO(dcramer): we should probably create the user if they dont
-        # exist, and ideally we'd also associate that with the event
-        euser = find_event_user(report, event)
-
-        if features.has("organizations:eventuser-from-snuba", project.organization) and event:
-            try:
-                find_and_compare_eventuser_data(event, euser)
-            except Exception:
-                logger.exception(
-                    "Error when attempting to compare EventUser with Snuba results & Event data"
-                )
+        euser = find_event_user(event, report)
 
         if euser and not euser.name and report.get("name"):
-            euser.update(name=report["name"])
```

Comment:
We should probably continue updating the event user name here as well. I think we should also add some stats to see how often people are actually using this - we're killing this functionality essentially.